fusesoc/main.py

```diff
@@ -198,6 +198,18 @@ def list_cores(cm, args):
         )
 
 
+def list_tools(cm, args):
+    from edalize.edatool import walk_tool_packages
+
+    _tp = list(walk_tool_packages())
+    maxlen = max(map(len, _tp))
+
+    for tool_name in _tp:
+        tool_class = get_edatool(tool_name)
+        desc = tool_class.get_doc(0)["description"]
+        print(f"{tool_name:{maxlen}} : {desc}")
```

Comment:
```suggestion
        print(f"{tool_name:{maxlen}}: {desc}")
```

Reply:
I wanted to have it like `fusesoc core list` which has a space between the longest name and the `:`